<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestión de Personal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0">
     <widget class="QTableView" name="necesidades_view">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Necesidades de personal:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Calendario</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="MiCalendario" name="calendarWidget">
          <property name="firstDayOfWeek">
           <enum>Qt::Monday</enum>
          </property>
          <property name="gridVisible">
           <bool>true</bool>
          </property>
          <property name="horizontalHeaderFormat">
           <enum>QCalendarWidget::ShortDayNames</enum>
          </property>
          <property name="verticalHeaderFormat">
           <enum>QCalendarWidget::NoVerticalHeader</enum>
          </property>
          <property name="navigationBarVisible">
           <bool>true</bool>
          </property>
          <property name="dateEditEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Listado</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTableView" name="bajas_view">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
        <item row="0" column="2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Filtrar por:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="filtro_cbox"/>
          </item>
          <item>
           <widget class="QLineEdit" name="buscar_ledit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="buscar_cbox"/>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuHerramientas">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionAnadir"/>
    <addaction name="actionEliminar"/>
    <addaction name="actionModificar"/>
   </widget>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionVista_previa"/>
    <addaction name="actionImprimir"/>
    <addaction name="separator"/>
    <addaction name="actionExportar_base_de_datos"/>
    <addaction name="actionImportar_base_de_datos"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuHerramientas"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImprimir"/>
   <addaction name="separator"/>
   <addaction name="actionAnadir"/>
   <addaction name="actionEliminar"/>
   <addaction name="actionModificar"/>
  </widget>
  <action name="actionAnadir">
   <property name="icon">
    <iconset>
     <normaloff>iconos/añadir.svg</normaloff>iconos/añadir.svg</iconset>
   </property>
   <property name="text">
    <string>Añadir</string>
   </property>
   <property name="toolTip">
    <string>Añadir nueva baja.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionEliminar">
   <property name="icon">
    <iconset>
     <normaloff>iconos/eliminar.svg</normaloff>iconos/eliminar.svg</iconset>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
   <property name="toolTip">
    <string>Eliminar una baja.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionImprimir">
   <property name="icon">
    <iconset>
     <normaloff>iconos/print.svg</normaloff>iconos/print.svg</iconset>
   </property>
   <property name="text">
    <string>Imprimir</string>
   </property>
   <property name="toolTip">
    <string>Imprime el mes en curso.</string>
   </property>
  </action>
  <action name="actionVista_previa">
   <property name="icon">
    <iconset>
     <normaloff>iconos/print-preview.svg</normaloff>iconos/print-preview.svg</iconset>
   </property>
   <property name="text">
    <string>Vista previa</string>
   </property>
   <property name="toolTip">
    <string>Vista previa de impresión</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionModificar">
   <property name="icon">
    <iconset>
     <normaloff>iconos/editar.svg</normaloff>iconos/editar.svg</iconset>
   </property>
   <property name="text">
    <string>Modificar</string>
   </property>
   <property name="toolTip">
    <string>Modificar fechas de la baja.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionExportar_base_de_datos">
   <property name="text">
    <string>Exportar base de datos</string>
   </property>
   <property name="toolTip">
    <string>Exporta base de datos a CSV</string>
   </property>
  </action>
  <action name="actionImportar_base_de_datos">
   <property name="text">
    <string>Importar base de datos</string>
   </property>
   <property name="toolTip">
    <string>Importar base de datos de CSV</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MiCalendario</class>
   <extends>QCalendarWidget</extends>
   <header>micalendario.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
